hemi/electrumx/electrumx.go

```diff
@@ -129,13 +129,12 @@ const (
 )
 
 // NewClient returns an initialised electrumx client.
-func NewClient(address string) (*Client, error) {
+func NewClient(address string, initialConns, maxConns int) (*Client, error) {
 	c := &Client{}
 
 	// The address may be empty during tests, ignore empty addresses.
 	if address != "" {
-		pool, err := newConnPool("tcp", address,
-			clientInitialConnections, clientMaximumConnections)
```

Comment:
can `clientInitialConnections` and `clientMaximumConnections` be removed completely?  or moved to the test file they are used in?

Reply:
They are currently still used in tests, although I can move them to the tests.